src/source/tile.js

```diff
@@ -91,6 +91,7 @@ class Tile {
     refreshedUponExpiration: boolean;
     reloadCallback: any;
     resourceTiming: ?Array<PerformanceResourceTiming>;
+    additionalRadius: number;
```

Comment:
Maybe rename to something like `renderedFeatureBuffer`

Reply:
"buffer" is like "layer" or "source" -- it means too many things in our code base! Maybe "padding" or "query padding"?

Reply:
Ugh you're so right